ember-cli-build.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const EmberApp = require('ember-cli/lib/broccoli/ember-app');
+const { Webpack } = require('@embroider/webpack');
```

Comment:
we should not be moving anything to the webpack version of embroider any more. If you need statiAppPaths then you should be using the ember-auto-import version of that. Let me know if you need more guidance on this

Reply:
I think your changes fixed the issue I was facing. now ember-auto-imports does it. 


Thank you :)